src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+function Sidebar() {
+  return (
+    <div class="video-drawer__container video__drawer container--sticky clr--secondary padding--medium">
```

Comment:
Change, `class=` to `className=`

Reply:
yes i have chnaged this in another branch, missed here